core/src/main/scala/eu/shiftforward/apso/json/ExtraJsonProtocol.scala

```diff
@@ -20,23 +21,25 @@ object ExtraJsonProtocol
   with ExtraMiscJsonProtocol
 
 trait ExtraTimeJsonProtocol {
+  def tryToParseDuration(duration: String): Try[FiniteDuration] =
```

Comment:
Can you make this private? It's strange to import `ExtraJsonProtocol` expecting only implicit instances and get a method like this in scope.

Reply:
Sure. It makes sense. I made it private in 1980bd5.